loggingservice/api/controllers/logs.py

```diff
@@ -58,4 +59,9 @@ def post():
         logging.exception(ex)
         raise OAuthProblem(ex)
 
-    return rs_handlers.success_response_only_status_code(200, "logging information successfully posted")
\ No newline at end of file
+    return rs_handlers.success_response_only_status_code(200, "logging information successfully posted")
+
+def version_search():
+    logs_yaml = open('logging.yaml')
+    parsed_logs_yaml = yaml.load(logs_yaml, Loader=yaml.FullLoader)
+    return parsed_logs_yaml['info']['version']
```

Comment:
It's a good practice to add a blank line at the end of a file. Please add that. Thanks.

Reply:
added